code/index.html

```diff
@@ -1,17 +1,84 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Business Site</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css" />
+
+    <link href="./transfonter.org-20240821-111631.zip" rel="stylesheet" />
   </head>
   <body>
-    <h1>Business name 🌻</h1>
+    <header>
+      <nav class="top-menu">
+        <ul class="nav-list">
+          <li>nomio</li>
+          <li>
+            <a href="https://pubmed.ncbi.nlm.nih.gov/37688976/">
+              Reducing lactic acid by 12% and increases endurance by 8% on
+              average
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- <video autoplay muted loop id="hero-video">
+      <source src="/run.mp4" type="video/mp4" />
+    </video> -->
```

Comment:
I think you forgot to change or delete this comment since you don’t have a video.